crates/gosub-bindings/Makefile

```diff
@@ -1,15 +1,16 @@
 # "global" compile settings
+CFLAGS_DEBUG := -std=c99 -g -Wall -Wextra -O0
+CFLAGS_RELEASE := -std=c99 -Wall -Wextra -O2
+CFLAGS := $(CFLAGS_RELEASE) # *** CONFIGURE RELEASE OR DUBUG MODE ***
+
 INCLUDE_DIR := include
 SRC_DIR := src
 CPPFLAGS := -I$(INCLUDE_DIR) -I$(INCLUDE_DIR)/nodes
-CFLAGS_DEBUG := -std=c99 -g -Wall -Wextra -O0
-CFLAGS_RELEASE := -std=c99 -Wall -Wextra -O2
-CFLAGS := $(CFLAGS_DEBUG)
 CC := gcc
 TARGET_DIR := ../../target/debug
```

Comment:
After some testing we also need a way to change this to `../../target/release` depending on the mode as well...